README.md

```diff
@@ -82,3 +82,39 @@ parsing messages is much slower than writing them, for example. This is
 because in order to parse messages it makes numerous dictionary lookups
 for AVP types, to be able to decode them. Encoding messages require less
 lookups and is generally simpler, thus faster.
+
+## Contribuite
+
+Before submitting PR, please execute `make test` to test your changes. 
+
+You will need to install `docker` and `docker-compose`
+
+```
+% make test
+Docker Compose is now in the Docker CLI, try `docker compose up`
```

Comment:
We shouldn't probably require installing docker. go-diameter is a 100% Go project & can potentially be tested in any environment which has Go.

go test ./..., etc. 

Also, make should not be required